roles/yoda_report/defaults/main.yml

```diff
@@ -0,0 +1,20 @@
+---
+# copyright Utrecht University
+
+irods_service_account: irods
+
+upload_priv_key: |
```

Comment:
Rename the variable to prevent conflicts with variables used in other roles.

Reply:
It would be nice to document this variable and include a warning indicating that it serves as a demo key. Always ensure it is replaced with a newly generated key.